manon/accordion.js

```diff
@@ -45,7 +45,7 @@ function initAccordion(accordion) {
     }
 
     // Set up the initial `aria-expanded` state.
-    if (button.getAttribute("aria-expanded") === "true") {
+    if (button.hasAttribute("aria-expanded")) {
```

Comment:
If I read the code correctly.

Now this just the checks a button has an `aria-expanded` attribute.

So that if the buttons do not have a `aria-expanded` attribute, the first one is expanded?

But still, what if you do not want the first one expanded?

Reply:
Then you can suppress that default behaviour by adding `aria-expanded=false` to the first item's button. Which is exactly what stopped working, and what this PR fixes.

Reply:
Ah ofcourse!